security/secure-markup.md

```diff
@@ -0,0 +1,52 @@
+# Secure Markup
+
+TODO: add TOC
+
+## Scope of this document
+Developers should be aware of the security implications of their front-end code. This document outlines some steps developers can take to reduce the risk of markup exposing us, or our users, to security flaws.
```

Comment:
"You should be aware...", "... your front end code", "steps you can take..." [Writing house style: `Address the user as "you" where possible.`]